lti_consumer/models.py

```diff
@@ -235,8 +242,10 @@ def clean(self):
             raise ValidationError({
                 "config_store": _("LTI Configuration stores on XBlock needs a block location set."),
             })
+        # If we're in an XBlock context, then check whether the database_config_enabled flag is enabled. Otherwise,
+        # don't restrict saving LtiConfiguration.
         if self.version == self.LTI_1P3 and self.config_store == self.CONFIG_ON_DB:
-            if not database_config_enabled(self.block.location.course_key):
+            if self.location and not database_config_enabled(self.block.location.course_key):
```

Comment:
I'm going to pull this into a separate pull request, but I needed it here to get it working.